### PR TITLE
Display placeholder for missing GH account & add who to contact for fixing info

### PR DESCRIPTION
**Pull request by matyasselmeci**
Hopefully the "GitHub: (none)" in people's contact info will serve as a hint to get people to send us their info?

### CODE REVIEW

File: src/templates/contacts.html.j2 [hunk 1]
```diff
@@ -36,12 +37,16 @@ OSG Contacts
...
       </div>
-      {% if user.GitHub -%}
       <div class="row">
         <div class="col-sm-3"><span class="font-weight-bold">GitHub:</span></div>
-        <div class="col-sm"><a href="https://github.com/{{user.GitHub}}">{{ user.GitHub }}</a></div>
+        <div class="col-sm">
+          {% if user.GitHub -%}
+          <a href="https://github.com/{{user.GitHub}}">{{ user.GitHub }}</a>
+          {%- else -%}
+          <span class="text-danger">(none)</span>
```
Comment (djw8605) [inline src/templates/contacts.html.j2]: Maybe make it explicit.  Like have the `(none)` text be:

    (none) Update github account name here

Or something more clever?  Bonus points if you make it a mailto:help@opensciencegrid.org with subject and some body already filled out.

Comment (matyasselmeci) [reply src/templates/contacts.html.j2]: That might be a little too glaring; we don't have GitHub info for 99% of our contacts.

**Comment by matyasselmeci** [comment]
Sure, that works. I couldn't add a link to the tooltip though.

**Comment by bbockelm** [comment]
I can confirm that if I'm not logged in, I don't see email addresses (but I can see them if I'm logged in).

How impossible is it to filter on people associated with an active resource, VO, or project?

**Comment by matyasselmeci** [comment]
Gee, doing that sort of correlation would be a lot easier if the data was in a SQL database or something :stuck_out_tongue:
It'd probably take me a day or two overall; I'll have to modify the data structures to keep track of contacts "seen." Hopefully it won't add too much to the processing time.

**Comment by djw8605** [comment]
I like it.  I say merge this now, and worry about filtering later.

**Comment by matyasselmeci** [comment]
Yeah, that should be a separate pull request anyway.

**Pull request merged**